src/edu/psu/sweng500/team8/gui/CombinationCell.java

```diff
@@ -229,16 +223,11 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
 	
 	private void buttonAction(ActionEvent actionEvent) {
 		JToggleButton button = (JToggleButton) actionEvent.getSource();
+		
 		if (button.isSelected()) {
-			gameSession.enterPencilMark(this.cell, Integer.parseInt(button.getText()), true);
-			
-			System.out.println("SELECTED " + button.getText());
-			System.out.println("CURRENT MARKS: " + this.cell.getPencilMarks());
+			gameSession.enterPencilMark(this.cell, Integer.parseInt(button.getText()), true);			

```

Comment:
condense this:

gameSession.enterPencilMark(this.cell, Integer.parseInt(button.getText()), button.isSelected());